src/contracts/Protocol/BountyQuestion.sol

```diff
@@ -3,13 +3,19 @@ pragma solidity 0.8.13;
 
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
+
+// Interfaces
+import "./interfaces/IQuestionAPI.sol";
+
+// Modifiers
 import "./modifiers/OnlyAPI.sol";
 import "./modifiers/OnlyStateController.sol";
+
+// Structs
 import "./Structs/QuestionData.sol";
-import "./interfaces/IBountyQuestion.sol";
 
 /// @custom:security-contact contracts@metricsdao.xyz
-contract BountyQuestion is IBountyQuestion, Ownable, OnlyApi, OnlyStateController {
```

Comment:
so you don't actually have to to implement the interface -- but other contracts will use the interface to call into this one (so that we don't have to include the entire file or write the ABI by hand)

If we don't do implement these interfaces, we need to be confident that a change in any "public" method signatures will be updated in other files (because the interface for the consumer could be out of sync with the implementation)

Reply:
That should be fine though right, we could just write some test coverage so that they'd fail if they go out of sync